### DISCUSSION TITLE
How to dynamically create Polyline GeoJSON?

### DISCUSSION THREAD

**Discussion by nonoumasy** [JustFly1984/react-google-maps-api#2969] [Q&A]
I would like to create a dynamic map like this using some Open data. Just wondering on how to do it. I'm thinking that the red lines  woud be some sort of geoJSON but how its created I'm not sure where to begin. 
Does anyone have any ideas?

Goal:
<img width="1440" alt="Screen Shot 2022-03-24 at 6 00 38 AM" src="https://user-images.githubusercontent.com/6312676/160260986-8b9b12b1-50f9-4dd3-9bc5-3a707d2687a3.png">

I tried to add it using the Polyline example but Polylines are continuous.
https://developers.google.com/maps/documentation/javascript/examples/polyline-simple?hl=en 

Here is what it looks like:
![Screen_Shot_2022-03-26_at_7 33 03_AM](https://user-images.githubusercontent.com/6312676/160261019-45b5fa8a-3346-48f8-b33a-dd0ccaf4cf3f.png)

Here is a regular version just using markers. It's not what I'm looking for though:
![Screen_Shot_2022-03-26_at_8 12 08_AM](https://user-images.githubusercontent.com/6312676/160261025-c371f79e-183f-492e-9244-eb1eb89896ef.png)

**Comment by EthanEFung** [comment]
What a cool project. Sounds like some sort of algorithm that given a lat lng on the map determines the closest adjacent point, and creates a polyline between them. However in order to get the clean lines, I don't think the points will be enough. Most likely you'll need to have the data of the road geometry.
